packages/_config/src/eslint.js

```diff
@@ -111,10 +111,13 @@ export const createConfig = async () =>
         'no-unused-vars': [
           WARN,
           {
-            args: 'after-used',
-            argsIgnorePattern: '^_',
-            ignoreRestSiblings: true,
-            varsIgnorePattern: '^ignored',
+            "args": "all",
+            "argsIgnorePattern": "^_",
+            "caughtErrors": "all",
+            "caughtErrorsIgnorePattern": "^_",
+            "destructuredArrayIgnorePattern": "^_",
+            "varsIgnorePattern": "^_",
+            "ignoreRestSiblings": true
```

Comment:
The rest are defaults and args should be after-used
```suggestion
            "args": "after-used",
            "caughtErrorsIgnorePattern": "^_",
            "varsIgnorePattern": "^_",
            "ignoreRestSiblings": true
```